immune_subtype_clustering/nf_launch.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import sys
+
+from orca.services.nextflowtower import NextflowTowerOps
+from orca.services.nextflowtower.models import LaunchInfo
+
+from utils import monitor_run, configure_logging
+
+
+def nextflow_execute_workflow(run_name: str, s3_file: str, cwl_file: str):
+    """Launches nf-synstage workflow in Nextflow Tower"""
+    ops = NextflowTowerOps()
+    info = LaunchInfo(
+        run_name=run_name,
+        pipeline="https://github.com/Sage-Bionetworks-Workflows/nf-cwl-wrap",
+        revision="main",
+        params={
+            "s3_file": s3_file,
+            "cwl_file": cwl_file,
+        },
+    )
+    run_id = ops.launch_workflow(info, "ondemand")
+    monitor_run(ops, run_id)
+
+
+def main():
+    run_name = sys.argv[1]  # "immune_subtype_clustering_execution"
```

Comment:
Nit: I'm not really a fan of `sys.argv` but for this purpose, I think it's fine.  If the intent is to make it into an Airflow DAG in the future, it's not worth the reworking.